.github/workflows/test.yml

```
@@ -0,0 +1,38 @@
+name: Run tests with docker compose
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: build pipenv
+        run: |
+          pip install pipenv
+          pipenv install
+        timeout-minutes: 3
+      - name: Setup DynamoDB Local
+        uses: rrainn/dynamodb-action@v2.0.0
+        with:
+          dbPath: # undefined by default, if this is undefined inMemory will be used
+          sharedDb: # undefined by default
+          delayTransientStatuses: # undefined by default
+          optimizeDbBeforeStartup: # undefined by default
+          port: 8000
+          cors: '*'
+      - name: tests
+        run: pipenv run python testApi.py
```

Consider [pytest](https://docs.pytest.org/en/6.2.x/).

It would let you run all the tests in one step and not have to add another step whenever you add more tests.
```suggestion
        run: pipenv run pytest
```

---

will look at this, I've never done unit testing in python so I wasn't sure how to set it up

---

will probably not move to this right now because I'm lazy